src/server.mjs

```diff
@@ -33,6 +33,9 @@ async function main(options) {
 
     server.use(express.static(wwwroot))
 
+    console.log("Debug information:")
```

Comment:
To distinguish from development "logs", I would prefer if [`process.stdout`][1] were used.

```js
process.stdout.write(`"Debug information:"
maxworkers: ", maxWorkers
)
```

I am in favor of adding _something_ to logging when the server starts, and I think this is a good start. 

@poef Is there any other information that is also relevant to output here?

[1]: https://nodejs.org/api/process.html#processstdout